app/services/metrics/review_turnaround/per_user_project.rb

```diff
@@ -10,22 +11,42 @@ def call
       private
 
       def process
-        filtered_reviews_ids.each_slice(BATCH_SIZE) do |batch|
-          Events::Review.eager_load(:review_request).find(batch).lazy.each do |review|
+        filtered_reviews_ids.lazy.each_slice(BATCH_SIZE) do |batch|
+          Events::Review.eager_load(:review_request).find(batch).each do |review|
```

Comment:
The JOIN statements should probably go here.

Reply:
Yep, you are right! I'm using the relations without the joins and eager_loads